MuseumsOfChicago/GeoBlazor.Sample.Museums/Components/Pages/Weather.razor

```diff
@@ -0,0 +1,64 @@
+﻿@page "/weather"
```

Comment:
Is there a reason this is included?  I assume it's a template file.  Just wondering if it still being included is an oversight